client/start.js

```diff
@@ -124,12 +124,15 @@ let pickComponentWrapper = null;
  * @returns {void}
  */
 export function navigateToPage(dispatch, path, doNotPushPath) {
+  const pathname =
+    !path || path === "undefined" ? global.location.pathname : path;
```

Comment:
can you add this to a function for better readability

Reply:
It just a condition and which is not used in multiple places. Adding functionality will make sense when you are using the same thing in multiple places.